cli/cmd/top.go

```diff
@@ -483,33 +492,70 @@ func processEvents(eventCh <-chan pb.TapEvent, requestCh chan<- topRequest, done
 	}
 }
 
-func pollInput(closing chan<- struct{}) {
+func pollInput(done chan<- struct{}, scrollLeft chan<- int, scrollRight chan<- int) {
 	for {
 		switch ev := termbox.PollEvent(); ev.Type {
 		case termbox.EventKey:
 			if ev.Ch == 'q' || ev.Key == termbox.KeyCtrlC {
-				closing <- struct{}{}
+				close(done)
 				return
 			}
+			if ev.Ch == 'a' || ev.Key==termbox.KeyArrowRight {
+				scrollLeft <- 1
+			}
+			if ev.Ch == 'd' || ev.Key==termbox.KeyArrowLeft  {
+				scrollRight <- 1
+			}
 		}
 	}
 }
 
-func renderTable(table *topTable, requestCh <-chan topRequest, done <-chan struct{}) {
-	ticker := time.NewTicker(100 * time.Millisecond)
-
+func renderTable(table *topTable, requestCh <-chan topRequest, done <-chan struct{}, scrollLeft <-chan int, scrollRight <-chan int) {
+	ticker := time.NewTicker(50 * time.Millisecond)
```

Comment:
Is this ticker change so that scrolling is responsive?  I think if we make the change I suggest below, this may not be necessary.

Reply:
I had done it to make it more responsive, but I feel after all the changes you've suggested the ticker will not be required to change. 👍